localstack/provision.sh

```diff
@@ -224,7 +224,7 @@ create_and_populate_activity_log_table() {
         ' \
       --region eu-west-2
 
-  npm install -g uuid
+  npm install -g uuid@10.0.0
```

Comment:
this fixes an issue I was having where the global uuid was intefering with the locally installed one (different verisons)